script/_dhcpv6_pd_ref

```diff
@@ -30,39 +30,110 @@
 #       This script manipulates DHCPv6-PD-REF configuration.
 #
 
-customise_dhcpcd_conf()
+DHCPCD_ENTER_HOOK="/etc/dhcpcd.enter-hook"
+DHCPCD_EXIT_HOOK="/etc/dhcpcd.exit-hook"
+
+PD_DAEMON_DIR="/opt/pd-daemon"
+PD_DAEMON_PATH="${PD_DAEMON_DIR}/dhcp6_pd_daemon.py"
+PD_DAEMON_SERVICE_NAME="dhcp6_pd_daemon.service"
+PD_DAEMON_SERVICE_PATH="/etc/systemd/system/${PD_DAEMON_SERVICE_NAME}"
+
+DHCP_CONFIG_FILE="/etc/dhcpcd.conf.orig"
+DHCP_CONFIG_ORIG_FILE="/etc/dhcpcd.conf.orig"
+DHCP_CONFIG_WITH_PD_FILE="/etc/dhcpcd.conf.with-pd"
+DHCP_CONFIG_NO_PD_FILE="/etc/dhcpcd.conf.no-pd"
```

Comment:
Personally I'd suggest to use `_PATH` instead of `_FILE` prefix for the full paths.